lib/controller.php

```diff
@@ -140,12 +140,14 @@ public function import_blob($blob) {
 		}
 
 		// If the blob sha already matches a post, then move on
+		// @TODO: check if we moved this post from one directory to another, so that we need to update the post type.
 		$id = $wpdb->get_var( "SELECT post_id FROM $wpdb->postmeta WHERE meta_key = '_sha' AND meta_value = '$blob->sha'" );
 		if ( $id ) {
 			WordPress_GitHub_Sync::write_log( __( 'Already synced blob ', WordPress_GitHub_Sync::$text_domain ) . $blob->path );
 			return;
 		}
 
+		$path = $blob->path;
```

Comment:
@mAAdhaTTah The property is there, is also used couple lines above on line 137. as soon the api get the blob from the api it does not return any path.